### PR TITLE
New docs

### PR DESCRIPTION
**Pull request by trigaten**

### CODE REVIEW

File: docs/source/environments/basalt.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+MineRL BASALT Competition Environments
+=======================================
+*The videos on this page do not represent the new MineRL environment or the new dataset.*
```
Comment (brandonhoughton) [inline docs/source/environments/basalt.rst]: Nit but this could be a warning tag

File: docs/source/index.rst [hunk 2]
```diff
@@ -31,6 +31,8 @@ MineRL: Towards AI in Minecraft
   :scale: 100 %
   :alt:
 
+*These videos do not represent the new MineRL environment or the new dataset.*
```
Comment (brandonhoughton) [inline docs/source/index.rst]: Here could also be a warning tag

File: docs/source/index.rst [hunk 3]
```diff
@@ -39,57 +41,37 @@ project and its related repositories and components!
...
+with the videogame Minecraft.
 
-  - :code:`minerl.env` -- A growing set of OpenAI Gym environments
-    in Minecraft. These environments leverage a **synchronous**, **stable**,
-    and **fast** fork of Microsoft Malmo called *MineRLEnv*.
-  - :code:`minerl.data` -- The main python module for ext with the *MineRL-v0* dataset
+Started as a research project at Carnegie Mellon University, MineRL aims to assist
+in the develpment of various aspects of artificial intelligence within Minecraft. 
+  
+We develop :code:`minerl` in our spare time, `please consider supporting us on Patreon`_
```
Comment (brandonhoughton) [inline docs/source/index.rst]: Let's remove the Patreon links - they don't actually go to support MineRL anymore

File: docs/source/environments/basalt.rst (hunk 1 above)
Comment (Miffyli) [reply docs/source/environments/basalt.rst]: Edit: mixed comments: this could be warning/note indeed.

File: docs/source/index.rst (hunk 2 above)
Comment (Miffyli) [reply docs/source/index.rst]: Tried to clean up thsi part but decided to remove it: it is bit too early in the docs to start throwing notes/warnings. They still serve as a nice visual abstract 🙃

**Review by Miffyli — APPROVED** [review]
Thanks @trigaten for the hard work! I have cleaned up things and removed parts we do not have time to clean up in immediate future, but lets add things for competition later :)

**Pull request merged**